package/gluon-core/luasrc/usr/lib/lua/gluon/migration.lua

```diff
@@ -0,0 +1,37 @@
+local unistd = require 'posix.unistd'
+
+
+local M = {}
+
+function M.get_migration_flag_path(migration_name)
+	return '/lib/gluon/core/migration/flags' .. migration_name
+end
+
+function M.is_migration_done(migration_name)
+	return unistd.access(M.get_migration_flag_path(migration_name))
+end
+
+function M.flag_migration_done(migration_name)
+	os.execute(M.get_migration_flag_path(migration_name))
+end
+
+function M.reinit_led_config(uci)
+	-- Remove LED configuration
+	uci:foreach('system', 'rssid', function(config)
+		uci:delete('system', config['.name'])
+	end)
+	uci:foreach('system', 'led', function(config)
+		uci:delete('system', config['.name'])
+	end)
```

Comment:
This should use `uci:delete_all()` - IIRC, deleting entries in a `uci:foreach()` is always wrong and can lead to all kinds of funky results, including segfaults (which should really be fixed, if it isn't by now...)